### ISSUE TITLE
Saved-User-Logins Not Functioning, Can't Find saved-refresh-tokens.json, and no userAuths section in config.yml

### ISSUE THREAD

**Issue by Ukrrus** [GeyserMC/Geyser#4936]
### Describe the bug

When using Geyser Standalone and going into config.yml, adding users to "saved-user-logins," those users still need to log in every time. I tried inputting their Bedrock username, Java username, and Microsoft account user with no luck. Log in is successful, Im able to connect (both on Bedrock and Java), but I need to log in every time. Im using playit.gg as a tunnel, but that doesn't seem to be causing any issues. 

I've seen on the wiki and other github issues that theres a "userAuth" section where you can add login details, but I don't see it on my config.yml file (just a nonfunctional saved-user-logins area) so Im worried that it got removed or isn't supported with Standalone. 

Im trying to set up a server for just me and my friend, who has a Java account but uses Bedrock until they can get a PC, so Im not worried about it being insecure or sharing info between us. I know you can get around this with Floodgate, but I tried and its a pain to install so Im hoping theres a way to fix this without resorting to downloading more programs. 

Wether theres a way to fix the bugs, or if theres an easy way around this, I would love to know the answer to my issues! 

### To Reproduce

1. Open and run GeyserStandalone
2. Go to File, open Geyser Folder, and then open config.yml
3. Navigate to "saved-user-logins:" and add usernames underneath, following example
4. Save file, relaunch server (and in my case, relaunch playit.gg), then relaunch GeyserStandalone
5. Go onto device with Bedrock. Join server, log in, and start playing
7. Disconnect from server, then reconnect. Forced to login multiple times, one after the other. 

### Expected behaviour

I expected to receive a token to reauthenticate. In GeyserMC wiki, I see that "The file that tokens will be saved in is in the same folder as this config, named saved-refresh-tokens.json", but I am unable to locate this file. 

Im new to MC & hosting servers, so Im not sure how you'd use this token if you did receive it, since you are prompted to log in with Microsoft and no option to input a token. 

Since this initial method isn't working, I'd also expect to find a "userAuth" section in config.yml so that I could try a new method. 

### Server Version and Plugins

Unrelated issue (maybe?), but I can't give /version results because Geyser Standalone responds with "Unknown command! Try /geyser help for a list of commands." no matter what I do, even if I type "/geyser help". Same with /geyser dump. 

The only thing the server has is the Vanilla Tweaks data pack. 

### Geyser Version

Geyser Standalone (unknown version, in the past few days)

### Minecraft: Bedrock Edition Device/Version

1.21.2, Ipad/IOS

**Comment by Ukrrus** [comment]
Forgot to add: Im self hosting on my macOS, server is otherwise totally functional.

**Comment by Camotoy** [comment]
userAuths has been removed from Geyser.

Please add a Geyser dump:

Run `geyser dump` (server console) / `/geyser dump` (ingame chat) on your Geyser instance or server, then copy the link it will give you and paste it here. A Geyser dump is a way of providing us with a bunch of useful information about your server and Geyser config. This doesn't include any IPs or sensitive data.

**Comment by Ukrrus** [comment]
> userAuths has been removed from Geyser.
> 
> Please add a Geyser dump:
> 
> Run `geyser dump` (server console) / `/geyser dump` (ingame chat) on your Geyser instance or server, then copy the link it will give you and paste it here. A Geyser dump is a way of providing us with a bunch of useful information about your server and Geyser config. This doesn't include any IPs or sensitive data.

I tried, with no luck (both on server console -in main folder & specific folder-, and in game chat)
Ill try again later today and screencap the results.

**Comment by Ukrrus** [comment]
> Make sure you're running the "geyser dump" command without the `/` infront of it - it's not needed when running commands in the server console.

That worked!! Thank you :D 
 https://dump.geysermc.org/avWCjgSs2i8Z2lpLivSCY6Nrew8RJJyk

**Comment by onebeastchris** [comment]
We updated the documentation - the file is not called `saved-refresh-tokens`, but `saved-auth-tokens`. Further, it seems like you did not any Bedrock usernames to the `saved-user-logins` section, which explains why their auth tokens were not saved.
In order for that feature to work, you need to add their Bedrock usernames to that config option, and it'll save the tokens from then on.

**Comment by Ukrrus** [comment]
I was having some issues with saved files not updating, will try that again now!

**Comment by Ukrrus** [comment]
It works!!! I reloaded geyser instead of quitting it and restarting.  I noticed when I quit Geyser Standalone that it regenerated the old config files, getting rid of my changes. Is this on purpose? Is there a way to avoid this, or should I just keep my own version and copy-paste it each time I launch Geyser?

I also can't find the saved-auth-tokens file (neither in hsperfdata file or in the rest of mac files), although it doesn't really matter as long as it works
